hack/images/kubeone-e2e/release.sh

```diff
@@ -16,7 +16,7 @@
 
 set -euox pipefail
 
-TAG=v0.1.20
+TAG=v0.1.27-go-1.18
```

Comment:
The image has been pushed manually. We don't have automation to push from the release branches.